src/components/visualizations/chart/highcharts/test/customConfiguration.spec.ts

```diff
@@ -621,6 +622,75 @@ describe('getCustomizedConfiguration', () => {
             const dataLabel = formatter.call(dataLabelPoint);
             expect(dataLabel).toBe(expectation);
         });
+
+        describe('percentage data label formatter', () => {
+
+            it('should return empty data labels with undefined percentage', () => {
+                const result = percentageDataLabelFormatter.call({});
+                expect(result).toBe('');
+            });
+
+            it('should format data labels to percentage for single axis chart implicitly', () => {
+                const dataLabel = {
+                    percentage: 55.55
+                };
+                const result = percentageDataLabelFormatter.call(dataLabel);
+                expect(result).toBe('55.55%');
+            });
+
+            it.each([
+                ['left', false],
+                ['right', true]
+            ])('should format data labels to percentage for %s single axis chart', (_: string, opposite: boolean) => {
+                const dataLabel = {
+                    percentage: 55.55,
+                    series: {
+                        chart: {
+                            yAxis: [{}]
+                        },
+                        yAxis: {
+                            opposite
+                        }
+                    }
+                };
+                const result = percentageDataLabelFormatter.call(dataLabel);
+                expect(result).toBe('55.55%');
+            });
+
+            it('should format data labels to percentage for dual axis chart on primary axis', () => {
+                const dataLabel = {
+                    percentage: 55.55,
+                    y: 123,
+                    series: {
+                        chart: {
+                            yAxis: [{}, {}]
+                        },
+                        yAxis: {
+                            opposite: false
+                        }
+                    }
+                };
+                const result = percentageDataLabelFormatter.call(dataLabel);
+                expect(result).toBe('55.55%');
+            });
+
+            it('should not format data labels to percentage for dual axis chart on secondary axis', () => {
```

Comment:
Consider to use data driven tests for this and the previous test

Reply:
Fixed